src/Microsoft.IdentityModel.Tokens/SecurityKey.cs

```diff
@@ -15,6 +17,7 @@ public abstract class SecurityKey
         private CryptoProviderFactory _cryptoProviderFactory;
         private object _internalIdLock = new object();
         private string _internalId;
+        private IDictionary<string, object> _propertyBag;
```

Comment:
This should be the concrete type instead of the interface for a private field.

Reply:
removed this code